apps/balancer-tools/src/components/Table.tsx

```diff
@@ -1,14 +1,25 @@
 import cn from "clsx";
+import { createContext, useContext } from "react";
+
+const TableContext = createContext({});
 
 const predefinedClasses = {
   gray: {
     solid: {
-      dark: { style: "bg-slate3", border: "border border-slate7 rounded" },
+      dark: { style: "bg-slate3", border: "border-0" },
+      darkWithBorder: {
```

Comment:
Why not simply add border to all?

Reply:
because on Internal Manager we use a blue table with no border 🤔